### ISSUE TITLE
Could not find package dir of analyzer

### ISSUE THREAD

**Issue by jonasfj** [dart-lang/pub-dev#3706]
From logs:
```
pub.analyzer.pana: Failed (v0.13.9+1) - dartrofit_adapter_rx/1.0.0-alpha01

Error:
    Invalid argument(s): Could not find package dir of analyzer

Stack:
    #0      _PubspecCache._packageDir (package:pana/src/tag_detection.dart:314:7)
    #1      _PubspecCache.pubspecOfPackage.<anonymous closure> (package:pana/src/tag_detection.dart:335:35)
    #2      _LinkedHashMapMixin.putIfAbsent (dart:collection-patch/compact_hash.dart:293:23)
    #3      _PubspecCache.pubspecOfPackage (package:pana/src/tag_detection.dart:334:26)
    #4      _DeclaredFlutterPlatformDetector._declaredFlutterPlatforms.<anonymous closure> (package:pana/src/tag_detection.dart:508:36)
    #5      _LinkedHashMapMixin.putIfAbsent (dart:collection-patch/compact_hash.dart:293:23)
    #6      _DeclaredFlutterPlatformDetector._declaredFlutterPlatforms (package:pana/src/tag_detection.dart:506:35)
    #7      new _PlatformViolationFinder.<anonymous closure> (package:pana/src/tag_detection.dart:551:20)
    #8      _PathFinder.findPath (package:pana/src/tag_detection.dart:134:34)
    #9      _PathFinder.findViolation (package:pana/src/tag_detection.dart:148:39)
    #10     _PlatformViolationFinder._findPlatformViolation (package:pana/src/tag_detection.dart:562:59)
    #11     Tagger.flutterPlatformTags (package:pana/src/tag_detection.dart:823:51)
    #12     PackageAnalyzer._inspect (package:pana/src/package_analyzer.dart:340:16)
    #13     StackZoneSpecification._registerUnaryCallback.<anonymous closure>.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:26)
    #14     StackZoneSpecification._run (package:stack_trace/src/stack_zone_specification.dart:209:15)
    #15     StackZoneSpecification._registerUnaryCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:14)
    #16     _rootRunUnary (dart:async/zone.dart:1192:38)
    #17     _CustomZone.runUnary (dart:async/zone.dart:1085:19)
    #18     _FutureListener.handleValue (dart:async/future_impl.dart:141:18)
    #19     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:682:45)
    #20     Future._propagateToListeners (dart:async/future_impl.dart:711:32)
    #21     Future._completeWithValue (dart:async/future_impl.dart:526:5)
    #22     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:36:15)
    #23     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:298:13)
    #24     PackageAnalyzer._pkgAnalyze (package:pana/src/package_analyzer.dart)
    #25     StackZoneSpecification._registerUnaryCallback.<anonymous closure>.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:26)
    #26     StackZoneSpecification._run (package:stack_trace/src/stack_zone_specification.dart:209:15)
    #27     StackZoneSpecification._registerUnaryCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:14)
    #28     _rootRunUnary (dart:async/zone.dart:1192:38)
    #29     _CustomZone.runUnary (dart:async/zone.dart:1085:19)
    #30     _FutureListener.handleValue (dart:async/future_impl.dart:141:18)
    #31     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:682:45)
    #32     Future._propagateToListeners (dart:async/future_impl.dart:711:32)
    #33     Future._completeWithValue (dart:async/future_impl.dart:526:5)
    #34     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:36:15)
    #35     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:298:13)
    #36     ToolEnvironment.runAnalyzer (package:pana/src/sdk_env.dart)
    #37     StackZoneSpecification._registerUnaryCallback.<anonymous closure>.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:26)
    #38     StackZoneSpecification._run (package:stack_trace/src/stack_zone_specification.dart:209:15)
    #39     StackZoneSpecification._registerUnaryCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:14)
    #40     _rootRunUnary (dart:async/zone.dart:1192:38)
    #41     _CustomZone.runUnary (dart:async/zone.dart:1085:19)
    #42     _FutureListener.handleValue (dart:async/future_impl.dart:141:18)
    #43     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:682:45)
    #44     Future._propagateToListeners (dart:async/future_impl.dart:711:32)
    #45     Future._completeWithValue (dart:async/future_impl.dart:526:5)
    #46     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:556:7)
    #47     StackZoneSpecification._run (package:stack_trace/src/stack_zone_specification.dart:209:15)
    #48     StackZoneSpecification._registerCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:119:48)
    #49     _rootRun (dart:async/zone.dart:1184:13)
    #50     _CustomZone.run (dart:async/zone.dart:1077:19)
    #51     _CustomZone.runGuarded (dart:async/zone.dart:979:7)
    #52     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1019:23)
    #53     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
    #54     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
    #55     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
    #56     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:169:5)
```

**Comment by jonasfj** [comment]
See https://github.com/dart-lang/pana/issues/675

**Comment by isoos** [comment]
This has been fixed in dartdoc 0.32.4 or earlier.
